private/buf/buffetch/internal/internal.go

```diff
@@ -157,8 +177,8 @@ type DirRef interface {
 }
 
 // NewDirRef returns a new DirRef.
-func NewDirRef(path string) (DirRef, error) {
-	return newDirRef("", path)
+func NewDirRef(format string, path string) (DirRef, error) {
```

Comment:
`format` is `dir`, why do we need to pass this? This seems incorrect unless I'm missing something